<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_N"/>
    <addaction name="action_O"/>
    <addaction name="separator"/>
    <addaction name="action_S"/>
    <addaction name="action_A"/>
    <addaction name="separator"/>
    <addaction name="action_P"/>
    <addaction name="separator"/>
    <addaction name="action_X"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
    <addaction name="separator"/>
    <addaction name="action_10"/>
   </widget>
   <widget class="QMenu" name="menu_T">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
    <addaction name="action_11"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_12"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_T"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_N"/>
   <addaction name="action_O"/>
   <addaction name="action_A"/>
   <addaction name="separator"/>
   <addaction name="action_4"/>
   <addaction name="action_5"/>
   <addaction name="action_6"/>
   <addaction name="action_7"/>
   <addaction name="action_8"/>
   <addaction name="action_10"/>
   <addaction name="separator"/>
   <addaction name="action_11"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>画图工具:</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>选择图型:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="shapeComboBox">
       <item>
        <property name="text">
         <string>无</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>直线</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>矩形</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>椭圆</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>画笔类型:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QComboBox" name="penStyleComboBox">
       <item>
        <property name="text">
         <string>实线</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>点线</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>画笔线宽:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QSpinBox" name="penWidthSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>画笔颜色:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QComboBox" name="penColorCombBox"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>填充颜色:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QComboBox" name="brushColorComboBox"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_N">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="statusTip">
    <string>新建文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_O">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="statusTip">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_S">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="statusTip">
    <string>保存文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_A">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/saveas.png</normaloff>:/images/saveas.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
   <property name="statusTip">
    <string>另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_P">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/print.png</normaloff>:/images/print.png</iconset>
   </property>
   <property name="text">
    <string>打印(&amp;P)</string>
   </property>
   <property name="statusTip">
    <string>打印文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_X">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="statusTip">
    <string>退出程序</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/zoom-in.png</normaloff>:/images/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="statusTip">
    <string>放大</string>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/zoom-out.png</normaloff>:/images/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>错小</string>
   </property>
   <property name="statusTip">
    <string>错小</string>
   </property>
  </action>
  <action name="action_6">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/zoom-1.png</normaloff>:/images/zoom-1.png</iconset>
   </property>
   <property name="text">
    <string>还原</string>
   </property>
   <property name="statusTip">
    <string>还原</string>
   </property>
  </action>
  <action name="action_7">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/rotate.png</normaloff>:/images/rotate.png</iconset>
   </property>
   <property name="text">
    <string>旋转</string>
   </property>
   <property name="statusTip">
    <string>旋转</string>
   </property>
  </action>
  <action name="action_8">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/shear.png</normaloff>:/images/shear.png</iconset>
   </property>
   <property name="text">
    <string>拉伸</string>
   </property>
   <property name="statusTip">
    <string>拉伸</string>
   </property>
  </action>
  <action name="action_10">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
   <property name="statusTip">
    <string>清空</string>
   </property>
  </action>
  <action name="action_11">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/tool.png</normaloff>:/images/tool.png</iconset>
   </property>
   <property name="text">
    <string>绘图工具栏</string>
   </property>
   <property name="statusTip">
    <string>绘图小程序</string>
   </property>
  </action>
  <action name="action_12">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/help.jpg</normaloff>:/images/help.jpg</iconset>
   </property>
   <property name="text">
    <string>关于本软件</string>
   </property>
   <property name="statusTip">
    <string>关于本软件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
